Userland/Libraries/LibJS/Bytecode/Op.cpp

```diff
@@ -1593,36 +1593,41 @@ DeprecatedString SetLocal::to_deprecated_string_impl(Bytecode::Executable const&
     return DeprecatedString::formatted("SetLocal {}", m_index);
 }
 
-DeprecatedString PutById::to_deprecated_string_impl(Bytecode::Executable const& executable) const
+static StringView property_kind_to_string(PropertyKind kind)
 {
-    auto kind = m_kind == PropertyKind::Getter
-        ? "getter"
-        : m_kind == PropertyKind::Setter
-        ? "setter"
-        : "property";
+    switch (kind) {
+    case PropertyKind::Getter:
+        return "getter"sv;
+    case PropertyKind::Setter:
+        return "setter"sv;
+    case PropertyKind::KeyValue:
+        return "key-value"sv;
+    case PropertyKind::DirectKeyValue:
+        return "direct-key-value"sv;
+    case PropertyKind::Spread:
+        return "spread"sv;
+    case PropertyKind::ProtoSetter:
+        return "proto-setter"sv;
+    default:
+        VERIFY_NOT_REACHED();
```

Comment:
If you remove the `default` and put this `VERIFY_NOT_REACHED()` at the end of the function outside the switch statement, we will get a compile error instead of a runtime crash if a new PropertyKind is added here